### ISSUE TITLE
Order in extract_geom()

### ISSUE THREAD

**Issue by glaroc** [appelmar/gdalcubes_R#66]
The FID column that is generated from extract_geom() seems to correspond to the row number of the sf features. However, the concept of "FID" doesn't really exist in sf, so maybe this could be clarified in the docs ?

**Comment by appelmar** [comment]
Agreed, maybe even an additional argument like `return_geometries = TRUE / FALSE` might be useful to do the merging automatically.
